posthog/urls.py

```diff
@@ -54,13 +54,24 @@ def login_view(request):
         email = request.POST["email"]
         password = request.POST["password"]
         user = cast(Optional[User], authenticate(request, email=email, password=password))
+        next_url = request.GET.get("next", "")
```

Comment:
Nit: You can just do `next_url = request.GET.get("next")`, if the param is not set, it'll default to `None`.